File: AoC2021.CSharp.Tests/Day0Tests.cs
namespace AoC2021.CSharp.Tests;

using Xunit;

public class Day10Tests {
    [Fact]
    public void SolvesExample() {
        const string ExampleInput = @"foo";

        var solver = new Day0();
        var actual = solver.Solve(ExampleInput);

        Assert.Equal("Day 0 C#: foo", actual);
    }

    [Fact]
    public void AlternativeSolvesExample() {
        const string ExampleInput = @"foo";

        var solver = new Day0Alternative();
        var actual = solver.Solve(ExampleInput);

        Assert.Equal("Day 0 C# Alternative: foo", actual);
    }
}

File: AoC2021.FSharp.Tests/Day0Tests.fs
module Day0Tests

open AoC2021.FSharp
open Xunit

[<Fact>]
let ``Solves Example`` () =
    let exampleInput = @"foo"

    let actual = Day0.Solve exampleInput
    Assert.Equal("Day 0 F#: foo", actual)
